bootloader/src/pyi_launch.c

```diff
@@ -415,60 +474,44 @@ pyi_launch_run_scripts(ARCHIVE_STATUS *status)
              * (Since we evaluate module-level code, which is not allowed to return an
              * object, the Python object returned is always None.) */
             if (!retval) {
+#if defined(WINDOWED) && defined(LAUNCH_DEBUG)
```

Comment:
I suggest to indent these preprocessor statements. this makes it easier to read.

Reply:
Hmm, okay, I've indented them to the level of the code. Is that better?

Reply:
I updated the code since I want to get this into 4.2 :-)